services/motion/builtin/builtin.go

```diff
@@ -105,12 +109,14 @@ func (ms *builtIn) Reconfigure(
 type builtIn struct {
 	resource.Named
 	resource.TriviallyCloseable
-	fsService       framesystem.Service
-	movementSensors map[resource.Name]movementsensor.MovementSensor
-	slamServices    map[resource.Name]slam.Service
-	components      map[resource.Name]resource.Resource
-	logger          golog.Logger
-	lock            sync.Mutex
+	backgroundWorkers sync.WaitGroup
```

Comment:
[q]: why not add `currentplan`? is this going to be handled in a subsequent ticket?

Reply:
I was going to leave this for a subsequent ticket as there is no need yet to surface it outside of the context of the MoveOnGlobe call until we expose plans to the user.  I could see an argument for doing it here though